### PR TITLE
Replace Travis CI with GitHub Actions

### PR DESCRIPTION
**Pull request by thtg88**
Hi there 👋
I was wondering if I could add a GitHub Actions workflow to run tests, so that I can later provide PHP 8 support for this package. Would this be of use? :)

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (tautvydasr) [inline .github/workflows/tests.yml]: Nitpick: could we remove empty line?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+name: tests
+
+on:
```
Comment (tautvydasr) [inline .github/workflows/tests.yml:3]: I am not an expert with GitHub actions but the current setup will run tests on every branch? I think it should run only on `master`:

```yml
on:
  push:
    branches: [master]
  pull_request:
    branches: [master]
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, mbstring
+          tools: composer:v2
+          coverage: none
+
+      - name: Install dependencies
```
Comment (tautvydasr) [inline .github/workflows/tests.yml:36]: Could we include composer validation step before installing dependencies?

```yml
    - name: Validate composer.json and composer.lock
      run: composer validate
```

**Review by tautvydasr — CHANGES_REQUESTED** [review]
Hey @thtg88, sure contributions are very much appreciated :+1: 

Currently, the repository is using travis to run tests so to switch to GitHub actions we should do the following:

1. Remove existing `.travis.yml` file
2. Add export ignore to `.gitattributes` for GitHub workflow config files
3. Update existing `Build Status` badge in the [README.md](https://github.com/tautvydasr/blade-class-names/blame/master/README.md#L3)

**Comment by thtg88** [comment]
Ah my bad! Sorry I had completely missed the repo had Travis CI integrated. I'm happy to address your feedback if you are happy with the migration to GitHub Actions, considering Travis is shutting down :)

Screenshot 2021-02-18 at 16 30 31

File: .github/workflows/tests.yml (hunk 1 above)
Comment (thtg88) [reply .github/workflows/tests.yml]: Removed ✅

File: .github/workflows/tests.yml (hunk 2 above)
Comment (thtg88) [reply .github/workflows/tests.yml:3]: Updated ✅

File: .github/workflows/tests.yml (hunk 3 above)
Comment (thtg88) [reply .github/workflows/tests.yml:36]: I've added that step in :)

**Comment by thtg88** [comment]
I've added the badge to the README, but I believe it will only be displayed once the PR has been merged to `master` as I'm not an owner of the repo :)

**Comment by thtg88** [comment]
I believe the feedback has been addressed - let me know what you think :)

**Review by tautvydasr — APPROVED** [review]
Thanks for addressing the feedback :+1:

**Pull request merged**